catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -12,24 +12,89 @@ spec:
   description: >-
     Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
   params:
-    - name: imageURL
-      description: URL of the main binary image.
+    - name: imageURLs
+      description: Space separated list of binary image URLs that are used to obtain SBOMs via cosign
       type: string
+    - name: containerImageIDs
+      description: Space separated list of Pyxis image IDs
+      type: string
+    - name: pyxisSecret
+      type: string
+      description: "The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert."
+    - name: server
+      type: string
+      description: The server type to use. Options are 'production' and 'stage'
+      default: production
   volumes:
     - name: workdir
       emptyDir: {}
   steps:
-    - name: download-sbom-for-image
-      # Image taken from https://github.com/sigstore/cosign
-      # and copied to Quay
+    - name: download-sbom-files
+      image:
+        quay.io/hacbs-release/release-utils:62f3c9ca85e45a4a76f2fb5c0ec9402f3cc08b2d
```

Comment:
I think when you change the other image line you should change this one too. I know it isn't required, but it would be confusing to have two different image tasks of the same image in one task

Reply:
OK, I'll do that.